src/snowcli/cli/sql.py

```diff
@@ -64,7 +64,7 @@ def execute_sql(
         readable=True,
         help="File to execute.",
     ),
-    connection: Optional[str] = typer.Option(
+    connection: str = typer.Option(
```

Comment:
I don't think I understand this change, especially seeing changes in `test_sql_overrides_connection_configuration`. Could you explain me that change?

Reply:
Good catch, there's not explicite need for connection if we have a default one and overrides.

Reply:
There is any reason why you removed optional only for one parameter and left for the others?